pkg/ebpf/c/bpf_endian.h

```diff
@@ -0,0 +1,113 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
Is this file copied from somewhere ?

Reply:
From `libbpf`, which is part of the kernel.